corehq/apps/users/forms.py

```diff
@@ -230,6 +231,11 @@ def direct_properties(self):
 
 
 class UpdateCommCareUserInfoForm(BaseUserInfoForm, UpdateUserRoleForm):
+    loadtest_factor = forms.IntegerField(
+        required=False, min_value=1, max_value=50000,
```

Comment:
50000! wow, that's big. I woudn't have though anything more than 10000 or even 1000 was necessary.